release/release_tests.yaml

```diff
@@ -4217,6 +4217,23 @@
     type: sdk_command
     file_manager: sdk
 
+- name: read_benchmark
```

Comment:
nit: The test name will need to incorporate it's 1node.

Reply:
how about we keep it as it is for now? we may want to extend later to cover multi-node environment. O.w. we may need to change the name again.

Reply:
Yeah we usually put the setup in name as well, e.g. "pipelined_training_50_gb" below, or "air_benchmark_xgboost_cpu_10".

Reply:
updated to `read_benchmark_single_node`.